fastapi/environment.yml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+  
+dependencies:
+  - fastapi
+  - pydantic==2.6.3
+  - spiceql
```

Comment:
do we want users to install the condo version of spiceql for development of the fastapi side? I feel it's better for the instructions to be more like "build spiceql locally" and add the fastapi stuff to the main env.yml.